HelpSource/Classes/PV_Copy.schelp

```diff
@@ -72,32 +71,51 @@ x.free;
 
 
 (
-s.waitForBoot({
-	b = Buffer.alloc(s,2048,1);
-	c = Buffer.alloc(s,2048,1);
-	d = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
-	e = Buffer.alloc(s,2048,1);
-	f = Buffer.alloc(s,2048,1);
-})
+
+b = Buffer.alloc(s, 2048, 1);
+c = Buffer.alloc(s, 2048, 1);
+e = Buffer.alloc(s, 2048, 1);
+f = Buffer.alloc(s, 2048, 1);
+
+d = Buffer.read(s, Platform.resourceDir +/+ "sounds/a11wlk01.wav");
```

Comment:
and same here - is it good style to add freeing code to code examples? genuine question, i don't have a preference one way or the other but it would be nice if this was consistent

Reply:
Ah yes. I've refactored this example, which now uses no interpreter variables anymore.